components/brave_ads/browser/ads_service_impl.cc

```diff
@@ -62,6 +62,8 @@
 
 namespace brave_ads {
 
+const static unsigned int _retries_count_on_network_change = 1;
```

Comment:
this is a constant so should follow `kRetriesCountOnNetworkChange` instead of `_retries_count_on_network_change`

Reply:
done